src/alphabase/bytes.cljc

```diff
@@ -138,3 +138,28 @@
             (- ai bi)))
         ; Reached the end of the shorter key, compare lengths.
         (- (alength a) (alength b))))))
+
+
+(defn copy-slice
+  "Copy a slice (defined by offset, length) from a byte array.
+
+  Omitting the slice `len` argument will copy remainder of
+  `src` array from offset (e.g, `(- (alength src) offset)` bytes)."
+  (^bytes [^bytes src offset len]
+   (let [dst (byte-array len)]
+     (copy src offset dst 0 len)
+     dst))
+  (^bytes [src offset]
+   (copy-slice src offset (- (alength ^bytes src) offset))))
+
+
+(defn concat-bytes
```

Comment:
Since this namespace has already established the pattern of shadowing some functions from `clojure.core`, how about just `concat` here?